core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -1,9 +1,9 @@
 <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
   <h3>Topic Similarities</h3>
-  <div>
+  <div class="oppia-topic-Similarities-container">
     Upload a csv file:<br>
     The first line of data should be a list of topic names. The next lines should be a symmetric adjacency matrix of similarities, which are values between 0 and 1.<br>
-    <input type="file" id="topicSimilaritiesFile">
+    <input type="file" id="topicSimilaritiesFile"><br>
```

Comment:
Don't use br tags for spacing. Use CSS rules instead.

Reply:
Done PTAL Thanks!